### DISCUSSION TITLE
Ability to Resize window, scales font and keeps rows/columns fixed

### DISCUSSION THREAD

**Discussion by jholickysr** [microsoft/terminal#14915] [Ideas]
There is a product that I currently use that provides the ability to fix the rows/columns but also allows you to resize the window and have the font scale dynamically. So picture a couple of Terminals running on your screen in 1/4 or 1/8th the size - but still looking exactly the same - just instantly resize and move it around.

I have tried with Terminal to replicate this but you have to manually us the +/- to adjust the content size. Terminal is working no different than say Excel - yes the window  decreases but the _internal_ content size remains the same - unless again you fidget with the zooming. I am looking for more of what VLC (media player) does - change the size of the window - and the contents reduce 1:1 in the same fashion - but **automatically**. 

At any rate, has anything like this been discussed as a possible future enhancement? 

thanks

**Comment by j4james** [comment]
There's already an open issue requesting something along the same lines (see #10466). They were suggesting that changing the font size should automatically resize the window (keeping the rows and columns fixed), while you're suggesting that changing the window size should automatically resize the font. In both cases the result is the same - the row/column dimensions remain unchanged. And I don't think there's any reason why both approaches couldn't be implemented at the same time.
